fastlane/lib/fastlane/actions/carthage.rb

```diff
@@ -31,6 +36,10 @@ def self.available_platforms
       end
 
       def self.available_options
+        default_options + options
```

Comment:
I was running into errors with the available_options size being too high on the Carthage class, so I simply split it into default_options, and options, which are concatenated in available_options. This is an okay workaround but maybe someone else has a better solution to abiding by the rubocop style enforcement?